daemon/Libraries/Config/AppConfig.cs

```diff
@@ -68,12 +68,13 @@ public class AppConfig
 
         // Versoning Information.
         public static string Version => "0.2.0";
+        public static string ReleaseChannel => "alpha";
```

Comment:
Why is this like this and actually committed? Please revert back to CHANNEL_PLACEHOLDER.

Reply:
Accidental commit from debugging.